compiler/llvm/llvmGlobalToWide.cpp

```diff
@@ -296,22 +296,13 @@ namespace {
     return load;
   }
 
-  void checkFunctionExistAndHasArgs(Value* f, Type* t, unsigned nArgs)
+  void checkFunctionExistAndHasArgs(Value* f, FunctionType* ft, unsigned nArgs)
```

Comment:
There is some ifdef'd code at the call sites to this that won't work anymore with this change. Fortunately, we now require LLVM 11, so you can just delete the ifdef for the old LLVM versions. Could you do that in this PR?

Reply:
Yes definitely, I missed those